src/raftstore/store/worker/region.rs

```diff
@@ -169,13 +171,14 @@ impl PendingDeleteRanges {
         ranges
     }
 
+    /// Removes and returns the peer info with the `start_key`.
     fn remove(&mut self, start_key: &[u8]) -> Option<(u64, Vec<u8>, Vec<u8>)> {
         self.ranges
             .remove(start_key)
             .map(|peer_info| (peer_info.region_id, start_key.to_owned(), peer_info.end_key))
     }
 
-    // before an insert is called, must call drain_overlap_ranges to clean the overlap range
+    /// Before an insert is called, it must call drain_overlap_ranges to clean the overlapped range.
```

Comment:
Looks like this is a requirement of the function, but what does this function do?